deb/usr/local/bin/ssl.sh

```diff
@@ -72,24 +76,14 @@ openssl req -new -nodes -key "${KEY}" -out "${CSR}" -config "${SSL_CONF}" -sha51
 echo "creating ca database"
 mkdir -p "${CA_DIR}/certs" "${CA_DIR}/newcerts"
 touch "${CA_DIR}/index.txt" "${CA_DIR}/.rand"
-date +%s > ${CA_DIR}/serial
+date +%s > "${CA_DIR}"/serial
 
 echo "signing request"
-openssl ca -batch -config "${SSL_CONF}" -passin pass:${PASSPHRASE} -policy policy_anything -out "${SIGNED}" -in "${CSR}" 2>/dev/null
+openssl ca -batch -config "${SSL_CONF}" -passin pass:"${PASSPHRASE}" -policy policy_anything -out "${SIGNED}" -in "${CSR}" -days $1
```

Comment:
What happens if $1 is not set? Is the command working with the `-days` parameter not getting a value?
If not, then I guess this is a breaking change and needs to be addressed in the changelog.